crates/sui-framework-build/src/compiled_package.rs

```diff
@@ -496,22 +496,23 @@ impl PackageHooks for SuiPackageHooks {
     }
 }
 
-enum AddressError {
-    NoPublishedAddress(Symbol),
-    InvalidPublishedAddress(Symbol, String),
+#[derive(PartialEq, Eq, PartialOrd, Ord)]
+pub enum UnpublishedDependency {
+    NoAddress(Symbol),
+    InvalidAddress(Symbol, String),
```

Comment:
Since we don't want to necessarily hard error for unpublished dependencies, I renamed this a bit based on new use.